app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb

```diff
@@ -15,6 +15,12 @@
               description: "Find out how to become a maths teacher, so you can show pupils how important maths is in everyday life and in understanding the world.",
               path: "/life-as-a-teacher/explore-subjects/maths"
             )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Languages",
+              description: "Become a languages teacher and help pupils recognise the importance of learning another language.",
```

Comment:
Is there any way of re-phrasing this -it's quite repetitive with `languages` 3 times?

Would it be worth mentioning some languages instead? Eg `Whether you teach French, Spanish, German, Mandarin...`

And/or talk about some aspect of languages - eg introducing pupils to other cultures?

(You can word something better than me!)